src/Microsoft.DotNet.VersionTools/Dependencies/BuildOutput/BuildDependencyInfo.cs

```diff
@@ -7,9 +7,9 @@
 using System.Collections.Generic;
 using System.Linq;
 
-namespace Microsoft.DotNet.VersionTools.Dependencies
+namespace Microsoft.DotNet.VersionTools.Dependencies.BuildOutput
 {
-    public class DependencyBuildInfo
+    public class BuildDependencyInfo : IDependencyInfo
```

Comment:
Note: These are breaking public API changes that will need to be consumed by anyone that is calling this library directly.

Reply:
Noted, the two known places are dotnet-docker-nightly and CLI.

Reply:
Didn't Roslyn also start using this library?

Reply:
It looks like [RoslynPublish](https://github.com/dotnet/roslyn/tree/3d57b165d1eb872389b60459493ebd583f5dc5f9/src/Tools/RoslynPublish) uses it to publish to dotnet/versions, but dependency changes are still done with [RepoUtil](https://github.com/dotnet/roslyn/tree/3d57b165d1eb872389b60459493ebd583f5dc5f9/src/Tools/RepoUtil).